jsonb/src/main/java/com/inrupt/client/jsonb/JsonbService.java

```diff
@@ -76,5 +84,28 @@ public <T> T fromJson(final InputStream input, final Type type) throws IOExcepti
             throw new IOException("Error parsing JSON", ex);
         }
     }
+
+    static class LowerCamelCase implements PropertyNamingStrategy {
```

Comment:
Do we not have a 'standard' implementation of this naming strategy?

By standard I mean one that we don't have to implement ourselves.